packages/client/src/index.ts

```diff
@@ -3,8 +3,10 @@ import * as t from "io-ts";
 import { fold } from "fp-ts/Either";
 
 import { UserProfile } from "@slicemachine/core/build/models";
-import { SharedSlice } from "@prismicio/types-internal/lib/customtypes/widgets/slices";
-import { CustomType } from "@prismicio/types-internal/lib/customtypes/CustomType";
+import {
+  CustomType,
+  SharedSlice,
+} from "@prismicio/types-internal/lib/customtypes";
```

Comment:
Did you check if this prevented tree-shaking?

Reply:
That's a good question. AFAIK, we don't check this at the moment.